consumer/consumer.go

```
@@ -82,19 +92,19 @@ func (c *DefaultIterator) consume() ([]Message, error) {
 	if c.consumer == nil {
 		cInst, err := q.createConsumerInstance()
 		if err != nil {
-			log.Printf("ERROR - creating consumer instance: %s", err.Error())
+			err = fmt.Errorf("Error creating consumer instance: [%v]", err)
```

why not just return the error instead of wrapping it?